### ISSUE TITLE
error trying to add the storyFormat

### ISSUE THREAD

**Issue by dcsan** [lazerwalker/twison#9]
![image](https://cloud.githubusercontent.com/assets/514002/11858857/444234cc-a41a-11e5-9f06-5238aeb7ec62.png)

I also got this error 

```js
The story format &ldquo;SugarCube&rdquo; could not be loaded (storyFormat was not called).
```
which is pointing to another format - SugarCube - which in fact *does* seem to be loaded for me.

but it seems you are calling that method:
http://github.com/lazerwalker/twison/raw/master/dist/format.js

```
window.storyFormat({
  "name": "Twison",
  "version": "0.0.1",
  "author": "Mike Lazer-Walker",
  "description": "Export your Twine 2 story as a JSON document",
  "proofing": false,
  "source": "<html>\r\n\t<head>\r\n\t\t .... ......."
});
```

**Comment by dcsan** [comment]
I was able to import this one
https://github.com/lazerwalker/twison

**Comment by dcsan** [comment]
FWIW it seems current version of twine has broken the format outputting anyway.

**Comment by lazerwalker** [comment]
Hmm. When I went to the web version of Twine 2, I got an error that it couldn't load my existing Twison format, but when I tried to re-add it, it worked fine.

What browser are you using?

**Comment by lazerwalker** [comment]
Aha, I'm able to reproduce your issue in Chrome (works fine in Firefox + Safari). So I hope you're using Chrome and that's what's going on?

**Comment by dcsan** [comment]
I'm using chrome. another thread here:
https://github.com/cauli/TwineJson/issues/4#issuecomment-165648355

these two projects are related?

**Comment by lazerwalker** [comment]
They're not explicitly related; I basically couldn't get TwineJson to work, and ended up writing this as a result of that (see https://github.com/cauli/TwineJson/issues/2). Similar techniques and approaches, different codebases.

**Comment by lazerwalker** [comment]
@dcsan Think I've figured this out. 

When trying to load external JS, Chrome (unlike Safari and Firefox) by default enforces MIME type checking to only allow execution of scripts that are of an executable type. Since GitHub doesn't expect you to use their raw file URLs as a CDN (slash probably actively wants to discourage it), they send all files served that way as "text/plain", which then sets off Chrome's security checks. Hosting this file elsewhere seems to clean up the issue for me.

tl;dr: try adding "http://lazerwalker.com/twison/format.js" as your story format instead. 

Let me know if that works?

**Comment by dcsan** [comment]
yep, that works!

so the main difference of your project vs cauli/TwineJson - seems yours is a bit more robust, solving cyclic dependencies etc?

**Comment by lazerwalker** [comment]
I'd characterize Twison as more focused. It's focused on providing first and foremost a basic dump of Twine's internal XML format into JSON, and then augmenting it with what data we can safely extract without introducing complexity like cyclic dependencies (e.g. parsing links). 

On top of that, I'd say there's an emphasis on documentation, stability, and ease of development. Things like up-to-date documentation on the expected output format, unit tests connected to a CI server, well-documented scripts to compile development versions of the script into production builds, etc.

**Comment by lazerwalker** [comment]
(Also, closing this issue for now, since it seems like the issue itself has been solved)
